apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/models/Sgv2TableAddRequest.java

```diff
@@ -5,82 +5,47 @@
 import javax.validation.constraints.NotBlank;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
-public class Sgv2TableAddRequest {
-  private String name;
-  private Sgv2Table.PrimaryKey primaryKey;
-
-  private List<Sgv2ColumnDefinition> columnDefinitions;
-
-  boolean ifNotExists;
-
-  Sgv2Table.TableOptions tableOptions = new Sgv2Table.TableOptions();
-
-  // For Deserializers
-  public Sgv2TableAddRequest() {}
-
-  public Sgv2TableAddRequest(String tableName) {
-    this.name = tableName;
-  }
-
-  @Schema(required = true, description = "The name of the table to add.")
-  @NotBlank(message = "TableAdd.name must be provided")
-  public String getName() {
-    return name;
-  }
-
-  public void setName(String name) {
+public record Sgv2TableAddRequest(
+    @Schema(
+            required = true,
+            description = "The name of the table to add.",
+            example = "cycling_events")
+        @NotBlank(message = "TableAdd.name must be provided")
+        String name,
+    @Schema(
+            required = true,
```

Comment:
if this is required you should add `@NotNull`

Reply:
Ok after trying it out realized that Update Table does not require primary key or columns; only create does.